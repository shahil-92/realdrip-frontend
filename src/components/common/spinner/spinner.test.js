import React from 'react';
import Spinner from '.';

describe('<Spinner />', () => {
  describe('Rendering', () => {
    it('should render nothing when display prop is not true', () => {
      const wrapper = shallow(<Spinner display={false} />);
      expect(wrapper).toMatchSnapshot();
    });
    it('should render correctly when display prop is true', () => {
      const wrapper = shallow(<Spinner display height={100} width={100} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});

