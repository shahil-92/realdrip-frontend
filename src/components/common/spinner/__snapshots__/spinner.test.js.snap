// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> Rendering should render correctly when display prop is true 1`] = `
<Loader
  className=""
  color="#0cd891"
  height={100}
  style={Object {}}
  timeout={0}
  type="TailSpin"
  visible={true}
  width={100}
/>
`;

exports[`<Spinner /> Rendering should render nothing when display prop is not true 1`] = `""`;
